kubernetes/clickhouse/values.yaml

```
@@ -0,0 +1,70 @@
+  TZ: Asia/Ho_Chi_Minh
+  akvorado:
+    enabled: false
+  clickhouse:
+    enabled: true
+    clusterName: clickhouse
+    replicaCount: 3
+    shardCount: 1
+    users:
+    - name: juniper
+      password: juniper@123
+      networks: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)